swagger2.0/oic.wk.res.swagger.json

```diff
@@ -182,6 +182,15 @@
         },
         "type": {
           "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/type"
+        },
+        "tag-pos-desc": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/tag-pos-desc"
+        },
+        "tag-rel-pos": {
+          "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/tag-rel-pos"
```

Comment:
Be sure to change this link to "tag-pos-rel" as well.

Reply:
Fixed

Reply:
Maybe I'm missing something but this reference still has the "pos" and "rel" inverted: "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties.core-schema.json#/definitions/tag-**_rel-pos_**"

Reply:
My bad, fixed now.